lib/rubycritic/rake_task.rb

```diff
@@ -0,0 +1,70 @@
+require 'rake'
+require 'rake/tasklib'
+require 'English'
+require 'rubycritic/cli/application'
+
+module Rubycritic
+  #
+  # A rake task that runs RubyCritic on a set of source files.
+  #
+  # This will create a task that can be run with:
+  #
+  #   rake rubycritic
+  #
+  # Example:
+  #
+  #   require 'rubycritic/rake_task'
+  #
+  #   RubyCritic::RakeTask.new do |task|
+  #     task.paths = FileList['lib/**/*.rb', 'spec/**/*.rb']
+  #   end
+  #
+  # :reek:TooManyStatements: { max_statements: 6 }
```

Comment:
Need to disable this warning via source code comment since "exclude" in config files doesn't work for detectors on class base. We just recently discovered [this bug](https://github.com/troessner/reek/issues/816) in Reek, I'll work on a fix soon.